README.md

```diff
@@ -88,8 +89,33 @@ parliament.parties('123').people.current.get
 parliament.people('123').letters('456').get
 ```
 
-#### What's #get?
-Once you've built your endpoint (`parliament.people.current`), we use the `#get` method to tell us you're finished building and ready to get the data. 
+### `#get`
+Once you've built your endpoint (`parliament.people.current`), we use the `#get` method to tell us you're ready to get the data.
+
+```ruby
+# target endpoint: 'http://test.com/people/123/letters/456.nt'
+response = parliament.people('123').letters('456').get #=> <#Parliament::Response [...]>
+
+response.each do |node|
+  # if your ntripile file contains a literal object it is stored into an instance variable accessible via the predicate
+  # name. For example, if you had a prediicate called 'http://id.ukpds.com/name' you would be able to access it like so:
```

Comment:
if we are going to use the code from the other day with linking objects using the predicate, then this would apply for other nodes as well as literals